### ISSUE TITLE
Look into a better alternative for transfer to Bianca

### ISSUE THREAD

**Issue by inaod568** [ScilifelabDataCentre/dds_cli#382]
Transferring to Bianca via transit works, but transit has an allocated amount of space for a specific project (defined when project created in SUPR), which may result in problems if we're working with huge amounts of data. 

@valyo Correct me if the above information is incorrect and if there's anything to elaborate on please fill in the gaps, thanks!

**Comment by valyo** [comment]
The information is in general correct, here are some details:

1. When they ssh to Transit, the users end up in their usual home directory on the non-sensitive HCP cluster(s).
2. The changes they made there are not persistent: if they download data from dds there, it will not persist beyond their ssh session
3. To download data in SENS (Bianca) project, they need to mount a specific directory of a SENS project on Transit, and then set the mount point as destination in the dds_cli command 
4. This directory is part of what is known as nobackup storage allocation of the SENS project

Basically, the users should be aware of (at least) three things:
a) to mount the correct SENS project on Transit
b) the size of the data they need to download vs the nobackup storage allocation of the corresponding SENS project
c) to start the download in a screen/tmux session for anything larger than a few hundreds of GB

**Comment by ewels** [comment]
Nice! How does a user mount the space? Is it something that could be done within the DDS CLI?

**Comment by valyo** [comment]
they mount it with a "mount_wharf sens_projectID" command, which prompts for a uppmax password + TOTP code.
IMO integration with SUPR and a solution like todays deliver would be the best option

**Comment by ewels** [comment]
I was half wondering if we could have a `--bianca-project` option (could even auto detect available projects from the user's groups and have a select menu) which then runs the mount command and sets the destination directory automatically.

The user would still need to do the Bianca auth but it would at least mean fewer steps to get right.

**Comment by ewels** [comment]
To be clear I think SUPR integration would be nice, I just worry that it will be technically very difficult to get it to work with the two separate authentication and decryption systems.

**Comment by inaod568** [comment]
> The information is in general correct, here are some details:
> 
> 1. When they ssh to Transit, the users end up in their usual home directory on the non-sensitive HCP cluster(s).
> 2. The changes they made there are not persistent: if they download data from dds there, it will not persist beyond their ssh session
> 3. To download data in SENS (Bianca) project, they need to mount a specific directory of a SENS project on Transit, and then set the mount point as destination in the dds_cli command
> 4. This directory is part of what is known as nobackup storage allocation of the SENS project
> 
> Basically, the users should be aware of (at least) three things: a) to mount the correct SENS project on Transit b) the size of the data they need to download vs the nobackup storage allocation of the corresponding SENS project c) to start the download in a screen/tmux session for anything larger than a few hundreds of GB

@valyo Is this correct?
1. `$ ssh -A <username>-<projid>@bianca.uppmax.uu.se`
2. Don't use `dds data get` get without the `--destination` option
3. `username@transit:~$ mount_wharf <sens_project>` -> `dds data get --destination <sens_project>/<destination>/`
4. No backup

**Comment by inaod568** [comment]
Great, thanks.

**Comment by inaod568** [comment]
Moving to Jira.
